pkg/llms/zhipuai/api.go

```diff
@@ -124,3 +124,34 @@ func (z *ZhiPuAI) SSEInvoke(params ModelParams, handler func(*sse.Event)) error
 	}
 	return Stream(url, token, params, ZHIPUAI_MODEL_Default_Timeout, nil)
 }
+
+func (z *ZhiPuAI) Validate() (string, error) {
+	url := BuildAPIURL(ZhiPuAILite, ZhiPuAIAsyncInvoke)
+	token, err := GenerateToken(z.apiKey, API_TOKEN_TTL_SECONDS)
+	if err != nil {
+		return "", err
+	}
+
+	testPrompt := []Prompt{
+		{
+			Role:    "user",
+			Content: "Hello!",
+		},
+	}
+
+	testParam := ModelParams{
+		Method:      ZhiPuAIAsyncInvoke,
+		Model:       ZhiPuAILite,
+		Temperature: 0.95,
+		TopP:        0.7,
+		Prompt:      testPrompt,
+	}
+
+	response, err := Post(url, token, testParam, ZHIPUAI_MODEL_Default_Timeout)
+	if err != nil {
+		return "", err
+	}
+	responseMsg := "TaskID:" + response.Data.TaskID
```

Comment:
Please return the full response at `status.conditionedStatus[0].Message`

Reply:
if the status code in reponse data is `200`,error can be nil here. Otherwise return a error indicates why this failed.